### DISCUSSION TITLE
EF water use characterization factors

### DISCUSSION THREAD

**Discussion by danielembc** [LCA-ActivityBrowser/activity-browser#1064] [Q&A]
Hi everyone, is there any explanation for you the EF water use characterization factors are not available in AB? Is there a way to include them? Thank you!

**Comment by marc-vdm** [comment]
We get the characterization factors from Brightway, you can best [ask here](https://github.com/brightway-lca/brightway2/issues)

Maybe this could also help you:
https://brightway.groups.io/g/AB-discussion/topic/94117759
